packages/shared/src/components/modals/NewSquadModal.tsx

```diff
@@ -77,6 +81,16 @@ function NewSquadModal({
     if (shouldQuit) onRequestClose();
     return null;
   };
+
+  if (isIntroShown) {
+    return (
+      <SquadsBetaModal
+        onRequestClose={handleClose}
+        onNext={() => setIsIntroShown(false)}
+      />
+    );
+  }
```

Comment:
Perhaps it's better to simply put the body as a conditional render on the modal?

Reply:
If it is going to be refactored, it can be converted to a step with a conditional to use it or not.